Vagrantfile

```diff
@@ -9,6 +9,9 @@ class Hash
   end unless Hash.method_defined?(:except)
 end
 
+# Specify minimum Vagrant version
+Vagrant.require_version '>= 2.2.10'
```

Comment:
This check will pass when 2.2.11 is released. Not sure we want it.

Reply:
@lioramilbaum Updated and lock to 2.2.10 now. All checks have passed. Please approve. Thanks.